src/state/selection/logics.ts

```diff
@@ -1,15 +1,15 @@
 import { createLogic } from "redux-logic";
 import { UIDisplayData } from "@aics/simularium-viewer";
 
-import { SET_COLOR_CHANGES } from "./constants";
 import { ReduxLogicDeps } from "../types";
-import { getAgentDisplayNamesAndStates } from "../trajectory/selectors";
-import { receiveAgentNamesAndStates } from "../trajectory/actions";
+import { getDefaultUIData } from "../trajectory/selectors";
+import { setDefaultUIData } from "../trajectory/actions";
+import { APPLY_USER_COLOR } from "./constants";
 
 const storeColorsLogic = createLogic({
     process(deps: ReduxLogicDeps, dispatch, done) {
         const { action, getState } = deps;
-        const uiData: UIDisplayData = getAgentDisplayNamesAndStates(getState());
+        const uiData: UIDisplayData = getDefaultUIData(getState());
```

Comment:
Minor concern about this name shortening from the true UIDisplayData to UIData in function names like getDefaultUIData.  It feels like the function is slightly lying to me.

Reply:
Ok! I'm fine to make the name more explicit